docsrc/users-guide/workflow.rst

```diff
@@ -145,8 +145,8 @@ An example of each is provided in the `next section <examples.rst>`__.
 
   It returns a :class:`CmdStanVB` object which contains an approximation the posterior distribution.
 
-* The :meth:`~CmdStanModel.optimize` runs one of
-  `Stan's optimization algorithms <https://mc-stan.org/docs/reference-manual/optimization-algorithms-chapter.html>`__.
+* The :meth:`~CmdStanModel.optimize` runs one of Stan's
+  `optimization algorithms <https://mc-stan.org/docs/reference-manual/optimization-algorithms-chapter.html>`__
```

Comment:
I'm not sure why you're escaping the name and link as code here.  Or in the original.  I'd just say 

```
runs one of Stan's *optimization algorithms* <https://mc-stan.org/docs/reference-manual/optimization-algorithms-chapter.html>.  
```

which renders as

> runs one of Stan's *optimization algorithms* <https://mc-stan.org/docs/reference-manual/optimization-algorithms-chapter.html>.

Or if you can use markdown for links, I'd just use

```
runs one of Stan's [*optimization algorithms*](https://mc-stan.org/docs/reference-manual/optimization-algorithms-chapter.html)
```

which renders as

> runs one of Stan's [*optimization algorithms*](https://mc-stan.org/docs/reference-manual/optimization-algorithms-chapter.html)

Reply:
This is how you structure a link in ReStructuredText

Reply:
But why the extra space before the periods?  Is that required?  This doc has two and the other PR from @WardBrian uses one.  I'd rather see none if that's legal!